.github/workflows/ci_verify_pr_contains_issue.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
   verify_linked_issue:
     runs-on: ubuntu-latest
     name: Ensure Pull Request has a linked issue.
-    if: github.event.pull_request.draft == false
+    if: github.event.pull_request.draft == false and github.event.label.name == 'enhancement'
```

Comment:
enhancement will cover up `bugfix`?

Reply:
Naa, It's still Draft and testing it out.

Reply:
oh okay! I saw everyone approved so thought maybe it's ready 😄